src/recovery.rs

```diff
@@ -454,6 +454,14 @@ impl Recovery {
     }
 
     fn on_packet_acked(&mut self, pkt_num: u64, epoch: packet::Epoch) -> bool {
+        // If the acked packet number is lower than the lowest unacked packet
+        // number it means that the packet is not newly acked, so return early.
+        if let Some(oldest) = self.sent[epoch].values().nth(0) {
+            if oldest.pkt_num > pkt_num {
+                return false;
+            }
```

Comment:
From the comment, I'm not sure if the intent is to exit early if `oldest.pkt_num == pkt_num`.

Perhaps renaming `oldest` to `lowest` and inverting the test logic would let the code line up with comment a bit more e.g.

```
if pkt_num < lowest.pkt_num {
  return false;
}
```

Reply:
Ok, done.